middle_end/flambda/from_lambda/prepare_lambda.ml

```diff
@@ -65,28 +65,6 @@ end
 (* CR-soon mshinwell: Remove mutable state *)
 let recursive_static_catches = ref Numbers.Int.Set.empty
```

Comment:
Could you also remove `recursive_static_catches` from `Prepare_lambda` in this PR ? I don't think it's used anymore.